src/button-dropdown/__tests__/button-dropdown-navigation.test.tsx

```diff
@@ -59,3 +62,31 @@ describe('Button dropdown header', () => {
     expect(findHeader(wrapper).getElement()).toHaveTextContent('Description');
   });
 });
+
+describe('Button dropdown navigate with mouse or keyboard', () => {
+  it('should add class is-focused if highligted with keyboard and remove when using mouse', () => {
+    const wrapper = renderInternalButtonDropdown({ items });
```

Comment:
Why internal button dropdown is rendered? Is it possible to render the one we export?